api/plugin_helpers.go

```diff
@@ -192,3 +228,28 @@ func VaultPluginTLSProviderContext(ctx context.Context, apiTLSConfig *TLSConfig)
 		return tlsConfig, nil
 	}
 }
+
+func GetSudoPaths() map[string]*regexp.Regexp {
```

Comment:
Tiny Nit: I don't think I've seen many cases where `Get` prefixes a "getter". For example, in a lot of the [client code](https://github.com/hashicorp/vault/blob/main/api/client.go#L689-L707), I see setters prefixed with `Set` and getters just the name of the item.